src/aktualizr_secondary/update_agent.h

```diff
@@ -11,8 +11,11 @@ class UpdateAgent {
   virtual bool isTargetSupported(const Uptane::Target& target) const = 0;
   virtual bool getInstalledImageInfo(Uptane::InstalledImageInfo& installed_image_info) const = 0;
 
-  virtual bool download(const Uptane::Target& target, const std::string& data) = 0;
+  //  virtual bool download(const Uptane::Target& target, const std::string& data) = 0;
+  //  virtual data::ResultCode::Numeric receiveData(const Uptane::Target& target, const uint8_t* data, size_t size) = 0;
```

Comment:
Are there plans here or can these be removed?

Reply:
It can be just removed.

Reply:
done